plugins/legacyoutput.md

```diff
@@ -8,7 +8,9 @@ keywords: legacyoutput filter content cms email flash
 
 The `legacyoutput` plugin changes TinyMCE's output, producing legacy elements such as `font`, `b`, `i`, `u`, `strike`, and use `align` attributes.
 
-This plugin can be useful if you want to use TinyMCE in an HTML mail client or to render contents to Flash.
+> **Important**: This plugin changes the editor content to HTML3. Using this plugin will greatly reduce the functionality of the editor.
```

Comment:
I feel like "This plugin changes the editor content to HTML3" maybe doesn't give enough details, but I'm not sure what else it should say.

Reply:
Also does it actually change the content? I thought it only created new content as HTML 3, but I'm not certain on that.

Reply:
Looking more, it looks like it really only just makes it so that formatting (eg bold, italic, etc...) will be applied using HTML 3 markup and that's really all it does.

Reply:
Fixed?